metric-anomaly-rule-api/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+import com.google.protobuf.gradle.generateProtoTasks
```

Comment:
can't this code go in the existing `metric-anomaly-detector-api` module?

Reply:
This will move to config-service later for CRUD, so keeping it separate.

Reply:
>  And also, this rule definition will be used by both the alert task producer + evaluator.

The `metric-anomaly-detector-api` module will be also be used by both the producer and evaluator modules?

Reply:
Yah, In that case, I will rename this package as `metric-anomaly-api` and host both AlertRule and AlertTask related to that package.

Reply:
As discussed offline, I will rename it to `metric-anomaly-data-model`  and will keep both together.